lib/robotremote.js

```diff
@@ -101,6 +102,11 @@ Server.prototype.getKeywordTags = function (name, response) {
     response(null, this.keywords[name].tags || []);
 };
 
+Server.prototype.getKeywordTypes = function (name, response) {
+    // https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#argument-types
+    response(null, this.keywords[name].types);
```

Comment:
what if types is not specified? how is that supposed to be filled? would be good and example keyword in test library where types are specified.